Kitodo/src/main/java/org/kitodo/production/forms/SearchResultForm.java

```diff
@@ -156,6 +158,22 @@ private void refreshFilteredList() {
         filteredList.addAll(resultList);
     }
 
+    /**
+     * delete process.
+     * @param processDTO to delete
+     */
+    public void delete(ProcessDTO processDTO){
+       try {
+           ProcessForm processForm = new ProcessForm();
+           processForm.setProcess(ServiceManager.getProcessService().getById(processDTO.getId()));
+           processForm.delete();
```

Comment:
Should method `delete` not inside `ProcessService` class so deleting a process is not depending on a form class?